scico/linop/radon_svmbir.py

```diff
@@ -30,39 +30,65 @@
     raise ImportError("Could not import svmbir; please install it.")
 
 
-class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam Radon transform based on svmbir.
+class TomographicProjector(LinearOperator):
+    r"""Tomographic projector based on svmbir.
 
     Perform tomographic projection of an image at specified angles, using
     the `svmbir <https://github.com/cabouman/svmbir>`_ package. The
     ``is_masked`` option selects whether a valid region for projections
     (pixels outside this region are ignored when performing the
     projection) is active. This region of validity is also respected by
     :meth:`.SVMBIRSquaredL2Loss.prox` when :class:`.SVMBIRSquaredL2Loss`
-    is initialized with a :class:`ParallelBeamProjector` with this option
+    is initialized with a :class:`TomographicProjector` with this option
     enabled.
+    Details about the underlying scanner geometries can be found
+    `here <https://svmbir.readthedocs.io/en/latest/overview.html>`_.
     """
 
     def __init__(
         self,
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
-        center_offset: Optional[float] = 0.0,
-        is_masked: Optional[bool] = False,
+        center_offset: float = 0.0,
+        is_masked: bool = False,
+        geometry: str = "parallel",
+        dist_source_detector: Optional[float] = None,
+        magnification: Optional[float] = None,
     ):
         """
         Args:
-            input_shape: Shape of the input array.
+            input_shape: Shape of the input array. Can be of length
+                either 2 (2D array) or 3 (3D array). When specifying
+                a 2D array, the format for the input_shape is
+                ``(num_rows,num_cols)``. For a 3D array, the format for
+                the input_shape is ``(num_slices,num_rows,num_cols)``.
+                Here ``num_slices`` denotes the number of slices in the
+                input, ``num_rows`` and ``num_cols`` denote the number
+                of rows and columns in a single slice of the input.
+                A slice is defined as the plane perpendicular to the
```

Comment:
Good to have slice defined here, but it would also help to explain how row and column fit into this geometry.

Reply:
I have put a basic description, but I am unsure about a more detailed description. I am not sure whether the rows or columns are oriented along X-rays. The svmbir documentation also seems to be missing this piece of information.

Is there anything else I could add here to make it clearer?

Reply:
Why do we use `(num_slices,num_rows,num_cols)` while `svmbir` uses `(num_views, num_slices, num_channels)`? I think their terminology is more standard for CT, and the axis names do have clearly defined meanings w.r.t. the imaging geometry.

Reply:
svmbir also uses the terminology `(num_slices,num_rows,num_cols)` for the reconstruction. The `(num_views, num_slices, num_channels)` terminology is used for the sinograms.

Reply:
OK, I didn't notice that.